samples/snippets/classification_boosted_tree_model_test.py

```diff
@@ -39,4 +39,27 @@ def test_boosted_tree_model(random_model_id: str) -> None:
     )
     del input_data["functional_weight"]
     # [END bigquery_dataframes_bqml_boosted_tree_prepare]
+    # [START bigquery_dataframes_bqml_boosted_tree_create]
+    # from sklearn.ensemble import GradientBoostingClassifier
+    import bigframes.ml.linear_model
+
+    # input_data is defined in an earlier step.
+    training_data = input_data[input_data["dataframe"] == "training"]
```

Comment:
No action needed, but something to consider for future: it would be nice to update the `prepare` section above to work without referencing an index (e.g. when ordering mode = "partial").

We have a few options, but the easiest will be to start with a string column and add `(True, "training")` as the last in the list of cases.

Aside: we have an issue open (349926559) to allow selecting _any_ column in the dataframe (such as `functional_weight`, which would be a natural choice in this example) even if its a different type, so long as a `True` (default) case is provided.